tensorflow_datasets/scripts/metadata_cleanup.py

```diff
@@ -0,0 +1,74 @@
+# Lint as: python3
+"""Removes all dataset versions from metadata directory which are not
+   present in registered versions of TFDS.
+
+To test:
+Goto tensorflow_datasets/scripts
+
+Args:
+dry_run : If given, it traverse and finds all
+         versions which are to be removed without
+         actually removing them. If not given
+         removes from metadata dir.
+```
+cd tensorflow_datasets/scripts
+
+For check not Remove : python metadata_cleanup.py --dry_run
+For Remove : python metadata_cleanup.py
+```
+"""
+import os
+from absl import flags
+from absl import app
+import tensorflow as tf
+from tensorflow_datasets.core import registered
+
+FLAGS = flags.FLAGS
+flags.DEFINE_boolean(
+    'dry_run', False, "Shows all versions\
+                      which are not registered")
+
+metadata_path = os.path.join(os.pardir, "testing/metadata")
+
+
+def _extract_metadata_versions(metadata_dir):
+  """Get all metadata direcotry versions paths,
+  It only extract the paths like 'dataset_name/version'
+  or 'dataset_name/config/versions' in metadata dir
+
+  Args:
+  metadata_dir : Path to metadat directory (testing/metadata)
```

Comment:
Could you try to format the docstring like the other files: http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

And don't forget the return sections